code/src/main/java/com/onlinecontacttracing/storage/data-storage/PositiveUser.java

```diff
@@ -73,7 +73,7 @@ public boolean userCanMakeMoreDraftsAfterBeingFlagged() {
 
   @Override
   public String toString() {
-    String person = "Negative User- ID:" + userId + ", email:" + userEmail + "\n";
+    String person = "Negative User- ID: " + userId + ", email: " + userEmail + "\n";
     String hasNumberOfAttemptsLeft = "  number of attempted drafts left: " + (Constants.NUMBER_OF_DRAFTS_ALLOWED - attemptedEmailDrafts) + "\n";
```

Comment:
I think handling all of this logic in toString() is the wrong place for it. If this is supposed to be something more essential than a toString() dump of the data it probably needs its own methods. At the very least, beware combining this toString() logic which could be called anywhere, with this accounting logic.

Reply:
All this code is from a separate PR, I might just undo the changes in these files because they are distracting from the main point: adding a servlet infrastructure